ParseLoginUI/build.gradle

```diff
@@ -5,6 +5,9 @@ dependencies {
     compile 'com.android.support:support-v4:22.0.0'
     compile 'com.parse:parse-android:1.10.1'
 
+    compile 'com.android.support:appcompat-v7:22.0.0'
+    compile 'com.android.support:recyclerview-v7:22.0.0'
+
```

Comment:
what target didn't compile? and i had a typo: why not leave this as `provided` and the sample/example `compile`?